src/tokens/functional/typography/typography.json

```diff
@@ -58,13 +59,16 @@
       },
       "lineHeight": {
         "large": {
-          "$value": "calc(48/32)"
+          "$value": 1.5,
+          "$type": "number"
         },
         "medium": {
-          "$value": "calc(32/20)"
+          "$value": 1.6,
```

Comment:
Slight shift in this one in the snapshot test, but thats probably because calc may resolve differently.. but it depends on the browser so they should be the same? Anyhow its so tiny I'm not sure we need to worry about it.

Reply:
But why, this is exactly `1.6`

Reply:
I don't know why, and the screenshots are taken with linux so I can't really test 😛 its so tiny, do you think we should just accept the small shift that _might_ happen?